pgjdbc/src/main/java/org/postgresql/core/Oid.java

```diff
@@ -76,6 +78,22 @@
   public static final int REF_CURSOR = 1790;
   public static final int REF_CURSOR_ARRAY = 2201;
 
+  private static final Map<Integer, String> OID_TO_NAME = new HashMap<Integer, String>();
+  private static final Map<String, Integer> NAME_TO_OID = new HashMap<String, Integer>();
```

Comment:
Could these be initialized to a better size?

Reply:
It could, however I'm inclined to treat that as over-engineering as it is initialization time only.